src/librbd/io/CopyupRequest.cc

```diff
@@ -175,11 +175,12 @@ void CopyupRequest<I>::read_from_parent() {
   if (m_image_ctx->enable_sparse_copyup) {
     ImageRequest<I>::aio_read(
       m_image_ctx->parent, comp, std::move(m_image_extents),
-      ReadResult{&m_copyup_extent_map, &m_copyup_data}, 0, m_trace);
+      ReadResult{&m_copyup_extent_map, &m_copyup_data},
+      m_image_ctx->op_flags, m_trace);
```

Comment:
Nit: unnecessary -- the op flags should only be applied in the `ObectReadRequest` already. Just update `ImageCtx::get_read_flags` appropriately.